addons/mass_mailing/static/src/js/wysiwyg.js

```diff
@@ -20,6 +20,29 @@ const MassMailingWysiwyg = Wysiwyg.extend({
         return res;
     },
 
+    /**
+     * @override
+     */
+     getValue: function () {
+        let editable = this._super(...arguments);
+        const parsedDocument = new DOMParser().parseFromString(editable, "text/html");
+        const dropableZone = parsedDocument.querySelector('.o_editable');
```

Comment:
IIRC the problem was about losing the dropzones and therefore nothing would work anymore. Could we change the condition so that it looks for the dropzone classes instead ? It would make more sense.